.circleci/config.yml

```diff
@@ -370,10 +370,6 @@ jobs:
           name: Restore GOPATH/pkg/mod
           keys:
             - influxdb-gomod-sum-{{ checksum "go.sum" }}
-      # NOTE: On go1.16 `go mod download` and `go mod tidy` no longer agree
-      # with each other. The Go team recommends using `go mod tidy` to download
-      # all the modules needed for build/test, so we check tidiness while we're
-      # at it. See https://github.com/golang/go/issues/43994#issuecomment-770053099
```

Comment:
1.17 restored the behavior of `go mod download` to match 1.15, but running `make checktidy` felt fine so I didn't bother reverting. Could be convinced to move `make checktidy` back to the `golint` job if others think it's a good idea.